### PR TITLE
Replace some utils with metals-languageclient

### PR DESCRIPTION
**Pull request by gabro**
Based on #37 (it will need a rebase)

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -86,14 +83,11 @@ function fetchAndLaunchMetals(context: ExtensionContext, javaHome: string) {
     ? serverVersionConfig
     : defaultServerVersion;
 
-  migrateStringSettingToArray("serverProperties");
-  migrateStringSettingToArray("customRepositories");
```
Comment (gabro) [inline src/index.ts]: these are not needed anymore, we're way past the migration

Comment (ckipp01) [reply src/index.ts]: Sounds good. To be honest, when I first put them in here I didn't know what they were for, and forgot to ask about them. So I'm glad they can go.

**Comment by gabro** [comment]
That's great! Thanks for testing @ckipp01! I'll remove the unused import as soon I'm back to a computer

File: src/index.ts (hunk 1 above)
Comment (gabro) [reply src/index.ts]: We used to have these config as comma separated values and we eventually migrated to proper arrays.

The chance that some was using those options *and* did not upgrade metals yet (considering it's auto updated) is so low that I think is safe to drop them

**Comment by gabro** [comment]
@ckipp01 should be ok now!

**Pull request merged**